pandas/_libs/tslibs/timezones.pyx

```diff
@@ -217,7 +217,7 @@ cdef object get_dst_info(object tz):
     if cache_key is None:
         # e.g. pytz.FixedOffset, matplotlib.dates._UTC,
         # psycopg2.tz.FixedOffsetTimezone
-        num = int(get_utcoffset(tz, None).total_seconds()) * 1000000000
+        num = int(get_utcoffset(tz, None).total_seconds()) * 1_000_000_000
```

Comment:
code check for cython hard-coded inters over 7ish characters?

Reply:
We don't have a code check for that, and it's not cython specific, it's just something I see here and there that I like to fix, would you be interested in a code check for that kind of thing?

Reply:
i think we can find better uses of time.